Components/Config.php

```diff
@@ -236,7 +240,7 @@ public function getBankTransferDueDateDays()
      *
      * @return boolean
      */
-    public function sendStatusMail()
+    public function sendPaymentStatusChangedMail()
```

Comment:
is this method name good enough. You return a bbolean if a mail should be sent. The method name means that a mail is sent here.